packages/features/bookings/components/event-meta/Members.tsx

```diff
@@ -18,6 +18,7 @@ export interface EventMembersProps {
   profile: BookerEvent["profile"];
   entity: BookerEvent["entity"];
   isPrivateLink: boolean;
+  hideProfileLink?: boolean;
```

Comment:
It doesn't have a "team" in the name because it is on entity and entity could be a team or user. Right now we set it only for team links